lessons/advanced/strings-in-depth.md

```diff
@@ -0,0 +1,163 @@
+---
+layout: page
+title: Strings in-depth 
+category: advanced
+order: 9
+lang: en
+---
+
+String manipulation is one of our must to have skills nowadays, ignoring the programming language. That's why we are including an in-depth on Strings.
+
+## Table of Contents
+
+- [Strings in Elixir](#strings-in-elixir)
+- [Char lists](#char-lists)
+- [Graphemes and Codepoints](#graphemes)
+- [String functions](#string-functions)
+  - [Length](#length)
+  - [Replace](#replace)
+  - [Duplicate](#duplicate)
+- [Exercises](#exercises)
+  - [Anagrams](#anagrams)
+
+## Strings in Elixir
+
+Elixir strings are nothing but a sequence of bytes. Let's watch a clear example:
+
+```elixir
+iex(2)> string = <<104,101,108,108,111>>
```

Comment:
@setzer777 these `iex` numbers show up in a number of examples, could you go through the lesson and correct all of them?